scipy/signal/tests/test_windows.py

```diff
@@ -213,5 +213,15 @@ def test_windowfunc_basics():
         window(6, *params, sym=False)
 
 
+def test_kaiser_derived():
+    M = 100
+    w = signal.kaiser_derived(M)
+
+    # Test for Princen-Bradley condition
+    assert np.allclose(w[:M//2] ** 2 + w[-M//2:] ** 2, 1.)
```

Comment:
Don't use a plain python `assert` in tests.  Instead, you can use `assert_allclose(w[:M//2] ** 2 + w[-M//2:] ** 2, 1.)` here.  (`assert_allclose` has already been imported from `numpy.testing`.)

Style nit: I think the general consensus among the scipy developers is to not put spaces around `**`.  So while you're editing this, also remove those spaces.